flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/catalog/BaseCatalog.java

```diff
@@ -77,7 +78,7 @@
  * The BaseCatalog that provides a default implementation for all methods in the {@link
  * org.apache.flink.table.catalog.Catalog} interface.
  */
-public abstract class BaseCatalog extends AbstractCatalog {
+public abstract class BaseCatalog extends AbstractCatalog implements TransformConverter {
```

Comment:
better to use a specific `transformConverter` like `propertiesConverter`?

Reply:
Done